staging/src/k8s.io/api/flowcontrol/v1beta1/types.go

```diff
@@ -57,6 +57,30 @@ const (
 	ResponseHeaderMatchedFlowSchemaUID                 = "X-Kubernetes-PF-FlowSchema-UID"
 )
 
+const (
+	// AutoUpdateAnnotationKey is the name of an annotation that enables automatic update of the spec
+	// of the suggested configuration object(s) if set to 'true'.
+	//
+	// On a fresh install, the configuration objects will have auto update enabled:
+	// apf.kubernetes.io/autoupdate-spec: 'true'
+	//
+	// On an existing cluster, during an upgrade kube-apiserver updates the suggested configuration objects when:
```

Comment:
I think it shouldn't happen only on kube-apiserver update. I would really like to avoid situation when kube-apiserver restart is a mitigation to some of the problems.
If auto-update feature is set, the I really think it should be happening periodically.

@MikeSpreitzer @lavalamp @deads2k - for thoughts too

Reply:
@tkashem - please update the comment here to reflect the current implementation

[also nit: please shorten the lines a bit]